Initial patch version for FoundryVTT 0.8.6 and newer

p4535992 opened the pull request:
NOTE: This is not operational version yet, need more test.
Hi everyone, I started with the help of [Szefo09](https://github.com/szefo09) a patching version for this module for the versions of FoundryVTT 0.8.6 and newer.
Unfortunately I got a little caught up in other things. If anyone wanted to help me especially with the tests I never find the time for, I would be really happy.

Commit:
Update for foundryvtt 0.8.8

p4535992 committed:
Update

gludington commented:
Thank you very much for this, @p4535992 .  Clearly, real life has kept me far away from being able to touch this project or even Foundry for long time - i have not even been able to install v0.8.x anywhere.  According to your note, do you have a set of ideas in mind that require further shakedown?

Commit:
Version 0.2.7 (maintenance mode)

p4535992 commented:
Hi gludington (happy to see you again) look I humbly updated the project and integrated the foundryvtt-types library most of the work (if not completely) was done by [szefo09](https://github.com/szefo09). I also couldn't find the time to test it properly (some feedback from the community has been there). **A little bit of everything would have to be redefined to integrate the new "Document" class of foundryvtt version 0.8.X**. For some details of what was done in the fork you can find here the details [changelog](https://github.com/p4535992/conditional-visibility/blob/master/CHANGELOG.md). Greetings.

Teshynil committed:
Update, Cleaning and levels compatibility

Some cleaning of some functions, implement a workaround for levels compatibility (Some testing pending), fix of pf2e sightlayer constructor, implement debouncing on pf2e

jllansford commented:
I just want to say I love this module and I do hope to get a fully supported version working soon.  If you guys need help testing I'd happily assist.  With that said as of right now I can't seem to get this branch to work at all in my game, though it might be some incapability with other mods.  I know it doesn't play well with the levels mod so I made sure to turn that off.

p4535992 committed:
Merge pull request #2 from Teshynil/master

Update, Cleaning and levels compatibility

Commit:
Preparation version 0.2.8

Commit:
Preparation version 0.2.8 part 2

Commit:
Preparation version 0.2.8 part 4

p4535992 committed:
Version 0.2.8

p4535992 commented:
> I just want to say I love this module and I do hope to get a fully supported version working soon. If you guys need help testing I'd happily assist. With that said as of right now I can't seem to get this branch to work at all in my game, though it might be some incapability with other mods. I know it doesn't play well with the levels mod so I made sure to turn that off.

Made a new patch version 0.2.8 , I don't guarantee anything, but you can test for it

ravingdragoon commented:
Oddly, for me, nothing works after version 0.2.1. I am running Foundry 0.8.9 on Chrome Version 92.0.4515.159 (Official Build) (64-bit) and took it down to just Conditional Visibility and its dependencies. After 0.2.1 t seems to recognize existing stealth/hidden states, but does not recognize newly added ones.

Commit:
[BETA] Add async and await on set/unset flag, must do some test

Commit:
[BETA] Some clean up

p4535992 commented:
> 
> 
> Oddly, for me, nothing works after version 0.2.1. I am running Foundry 0.8.9 on Chrome Version 92.0.4515.159 (Official Build) (64-bit) and took it down to just Conditional Visibility and its dependencies. After 0.2.1 t seems to recognize existing stealth/hidden states, but does not recognize newly added ones.

Can you give me more information? Are there any exceptions on the web console? Can you give me a "step to step" description to replicate the problem?

Commit:
Update README.md

Teshynil committed:
Cleaning, formatting and bug corrections

- Cleaning of some functions and use of the constants added by Marco Tenti
- Remove deprecations for v9 of foundry
- Expand in the workaround for levels
- Fixing hidden status and vision for linked actors and non representing tokens
- Added a variable to the tokens for future usage in compatibility layers with other modules

p4535992 committed:
Merge pull request #3 from Teshynil/master

Cleaning, formatting and bug corrections

p4535992 committed:
Version 0.2.9

p4535992 commented:
hey @gludington , it seems that @Teshynil  has made a stable version of the fork, see if you want to test it and possibly try to merge this PR.

p4535992 committed:
Little fix

p4535992 committed:
Little fix

ravingdragoon commented:
@p4535992 sorry it took so long. After troubleshooting it seems that what is not working properly for me after the 0.2.9 from @Teshynil is the Auto-Hide on Stealth feature for DnD 5e and the show/request stealth roll when applying Hidden through the status effects. I do not get any errors in the console when I try to roll stealth, but I do get the following warning when I use the feature through the Status Effects menu instead of using a Stealth roll.

dae |  Dae macro  on   
Actor5e {overrides: {…}, _preparationWarnings: Array(0), armor: null, shield: null, _classes: {…}, …}  
apps: {}  
armor: null  
getOwnedItem: ƒ (id)  
labels: {creatureType: 'Humanoid (Elf)'}  
longRest: async ƒ ()  
overrides: {data: {…}, flags: {…}}  
shield: null  
shortRest: async ƒ ()  
_classes: {}  
_lastWildcard: null  
_preparationWarnings: []  
_sheet: null  
_tokenImages: (2) ['https://assets.forge-vtt.com/604155e2aca674c278836…or-cos/characters/104_-_Rahadin/RahadinAction.png', 'https://assets.forge-vtt.com/604155e2aca674c278836…ules/cor-cos/characters/104_-_Rahadin/avatar.webp']  
applyActiveEffects: (...)  
applyDamage: (...)  
classes: (...)  
collection: (...)  
collectionName: (...)  
compendium: (...)  
data: ActorData {_id: 'M2NlY2IzOWI5Nzhi', name: 'Rahadin', type: 'npc', img: 'https://assets.forge-vtt.com/604155e2aca674c278836…rse-of-strahd/characters/139_-_Rahadin/avatar.png', data: {…}, …}  
documentName: (...)  
effects: (...)  
entity: (...)  
folder: (...)  
getRollData: (...)  
hasPlayerOwner: (...)  
id: (...)  
img: (...)  
isEmbedded: (...)  
isOwner: (...)  
isPolymorphed: (...)  
isToken: (...)  
itemTypes: (...)  
items: (...)  
limited: (...)  
link: (...)  
name: (...)  
owner: (...)  
pack: null  
parent: null  
permission: (...)  
prepareData: (...)  
rollAbilitySave: (...)  
rollAbilityTest: (...)  
rollDeathSave: (...)  
rollSkill: (...)  
sheet: (...)  
temporaryEffects: (...)  
toCompendium: (...)  
token: (...)  
type: (...)  
uuid: (...)  
visible: (...)  
_data: (...)  
_id: (...)  
[[Prototype]]: Actor  
[ActiveEffectData]  
0: ActiveEffectData {_id: 'qkpo38nmr3ad786g', changes: Array(0), disabled: false, duration: EffectDurationData, icon: 'modules/conditional-visibility/icons/newspaper.svg', …}  
length: 1  
[[Prototype]]: Array(0)  
{}  
[[Prototype]]: Object  
[[Prototype]]: Object  
warn	@	dae.js:24  
daeMacro	@	dae.js:1069  
_preCreateActiveEffect	@	dae.js:860  
🎁call_wrapper	@	libWrapper-wrapper.js:576  
🎁CONFIG.ActiveEffect.documentClass.prototype._preCreate#0	@	libWrapper-wrapper.js:164  
_preCreateDocumentArray	@	foundry.js:8669  
_createEmbeddedDocuments	@	foundry.js:8629  
create	@	backend.mjs:95  
async function (async)  
create	@	backend.mjs:93  
createDocuments	@	document.mjs:332  
create	@	document.mjs:431  
update	@	api.js:386  
changeIconCounter	@	keybinds.js:215  
onEffectClick	@	keybinds.js:115  
dispatch	@	jquery.min.js:2  
v.handle	@	jquery.min.js:2

Commit:
Added github workflow for release

Commit:
Update package.json

Commit:
Update tsconfig.json

p4535992 committed:
Version 0.2.10

p4535992 commented:
@ravingdragoon  can you try version 0.2.10 ? i made some small fix.

p4535992 committed:
Small bug fix

p4535992 committed:
update CHANGELOG.md

ravingdragoon commented:
@p4535992 I tried 0.2.10 and 0.2.11 and neither works fully. It looks like the only remaining problem is that the hidden state is not being auto-applied after the stealth roll. If I go to the hidden condition in my HUD it says the current stealth value (same as most recent stealth roll) and when I manually apply the hidden state conditional visibility seems to function properly, the status is even removed properly when the hidden token makes an attack. Again, I don't get any errors, but I do get the following warnings:

dae.js:400 dae | you are using dice expressions in a numeric field this will be disabled when foundry 0.9 is released  
applyDaeEffects @ dae.js:400  
applyBaseActiveEffects @ dae.js:348  
🎁call_wrapper @ libWrapper-wrapper.js:551  
🎁CONFIG.Actor.documentClass.prototype.applyActiveEffects#0 @ libWrapper-wrapper.js:164  
prepareEmbeddedEntities @ foundry.js:12397  
prepareData @ foundry.js:9413  
prepareData @ entity.js:55  
🎁call_wrapped @ libWrapper-wrapper.js:467  
patchedPrepareData @ skl-customization.js:16  
🎁call_wrapper @ libWrapper-wrapper.js:576  
prepareData @ dae.js:749  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.documentClass.prototype.prepareData#0 @ libWrapper-wrapper.js:164  
_postUpdateDocumentCallbacks @ foundry.js:8984  
_handleUpdateDocuments @ foundry.js:8914  
_updateDocuments @ foundry.js:8803  
async function (async)  
_updateDocuments @ foundry.js:8794  
update @ backend.mjs:154  
async function (async)  
update @ backend.mjs:151  
updateDocuments @ document.mjs:366  
update @ document.mjs:448  
callOriginalFunction @ Monkey.js:80  
(anonymous) @ patches.js:100  
setFlag @ document.mjs:596  
(anonymous) @ ConditionalVisibilityFacade.js:131  
hide @ ConditionalVisibilityFacade.js:110  
(anonymous) @ ConditionalVisibilitySystem5e.js:117  
_call @ VM9225:4  
callAll @ foundry.js:153  
callback @ foundry.js:8777  
(anonymous) @ foundry.js:8716  
_handleCreateDocuments @ foundry.js:8716  
_createDocuments @ foundry.js:8612  
async function (async)  
_createDocuments @ foundry.js:8603  
create @ backend.mjs:96  
async function (async)  
create @ backend.mjs:93  
createDocuments @ document.mjs:332  
create @ document.mjs:431  
toMessage @ foundry.js:6128  
toMessage @ d20-roll.js:122  
doRollSkill @ patching.js:114  
async function (async)  
doRollSkill @ patching.js:108  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.entityClass.prototype.rollSkill#0 @ libWrapper-wrapper.js:164  
rollSkillMacro @ bundle.min.js:1  
_handleMacros @ bundle.min.js:1  
doHandleActionEvent @ bundle.min.js:1  
handleActionEvent @ bundle.min.js:1  
handleClick @ bundle.min.js:1  
(anonymous) @ bundle.min.js:1  
dispatch @ jquery.min.js:2  
v.handle @ jquery.min.js:2  

14:12:37.026 dae.js:401 Actor Zion Actor.F85ubwNpBPJxuKNQ Change is data.attributes.movement.walk:   
applyDaeEffects @ dae.js:401  
applyBaseActiveEffects @ dae.js:348  
🎁call_wrapper @ libWrapper-wrapper.js:551  
🎁CONFIG.Actor.documentClass.prototype.applyActiveEffects#0 @ libWrapper-wrapper.js:164  
prepareEmbeddedEntities @ foundry.js:12397  
prepareData @ foundry.js:9413  
prepareData @ entity.js:55  
🎁call_wrapped @ libWrapper-wrapper.js:467  
patchedPrepareData @ skl-customization.js:16  
🎁call_wrapper @ libWrapper-wrapper.js:576  
prepareData @ dae.js:749  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.documentClass.prototype.prepareData#0 @ libWrapper-wrapper.js:164  
_postUpdateDocumentCallbacks @ foundry.js:8984  
_handleUpdateDocuments @ foundry.js:8914  
_updateDocuments @ foundry.js:8803  
async function (async)  
_updateDocuments @ foundry.js:8794  
update @ backend.mjs:154  
async function (async)  
update @ backend.mjs:151  
updateDocuments @ document.mjs:366  
update @ document.mjs:448  
callOriginalFunction @ Monkey.js:80  
(anonymous) @ patches.js:100  
setFlag @ document.mjs:596  
(anonymous) @ ConditionalVisibilityFacade.js:131  
hide @ ConditionalVisibilityFacade.js:110  
(anonymous) @ ConditionalVisibilitySystem5e.js:117  
_call @ VM9225:4  
callAll @ foundry.js:153  
callback @ foundry.js:8777  
(anonymous) @ foundry.js:8716  
_handleCreateDocuments @ foundry.js:8716  
_createDocuments @ foundry.js:8612  
async function (async)  
_createDocuments @ foundry.js:8603  
create @ backend.mjs:96  
async function (async)  
create @ backend.mjs:93  
createDocuments @ document.mjs:332  
create @ document.mjs:431  
toMessage @ foundry.js:6128  
toMessage @ d20-roll.js:122  
doRollSkill @ patching.js:114  
async function (async)  
doRollSkill @ patching.js:108  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.entityClass.prototype.rollSkill#0 @ libWrapper-wrapper.js:164  
rollSkillMacro @ bundle.min.js:1  
_handleMacros @ bundle.min.js:1  
doHandleActionEvent @ bundle.min.js:1  
handleActionEvent @ bundle.min.js:1  
handleClick @ bundle.min.js:1  
(anonymous) @ bundle.min.js:1  
dispatch @ jquery.min.js:2  
v.handle @ jquery.min.js:2  

14:12:37.026 dae.js:406 change value calculation failed for Actor5e {overrides: {…}, _preparationWarnings: Array(0), armor: null, shield: null, _classes: {…}, …} {key: 'data.attributes.movement.walk', value: '', mode: 2, priority: 9, effect: ActiveEffect5e}  
applyDaeEffects @ dae.js:406  
applyBaseActiveEffects @ dae.js:348  
🎁call_wrapper @ libWrapper-wrapper.js:551  
🎁CONFIG.Actor.documentClass.prototype.applyActiveEffects#0 @ libWrapper-wrapper.js:164  
prepareEmbeddedEntities @ foundry.js:12397  
prepareData @ foundry.js:9413  
prepareData @ entity.js:55  
🎁call_wrapped @ libWrapper-wrapper.js:467  
patchedPrepareData @ skl-customization.js:16  
🎁call_wrapper @ libWrapper-wrapper.js:576  
prepareData @ dae.js:749  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.documentClass.prototype.prepareData#0 @ libWrapper-wrapper.js:164  
_postUpdateDocumentCallbacks @ foundry.js:8984  
_handleUpdateDocuments @ foundry.js:8914  
_updateDocuments @ foundry.js:8803  
async function (async)  
_updateDocuments @ foundry.js:8794  
update @ backend.mjs:154  
async function (async)  
update @ backend.mjs:151  
updateDocuments @ document.mjs:366  
update @ document.mjs:448  
callOriginalFunction @ Monkey.js:80  
(anonymous) @ patches.js:100  
setFlag @ document.mjs:596  
(anonymous) @ ConditionalVisibilityFacade.js:131  
hide @ ConditionalVisibilityFacade.js:110  
(anonymous) @ ConditionalVisibilitySystem5e.js:117  
_call @ VM9225:4  
callAll @ foundry.js:153  
callback @ foundry.js:8777  
(anonymous) @ foundry.js:8716  
_handleCreateDocuments @ foundry.js:8716  
_createDocuments @ foundry.js:8612  
async function (async)  
_createDocuments @ foundry.js:8603  
create @ backend.mjs:96  
async function (async)  
create @ backend.mjs:93  
createDocuments @ document.mjs:332  
create @ document.mjs:431  
toMessage @ foundry.js:6128  
toMessage @ d20-roll.js:122  
doRollSkill @ patching.js:114  
async function (async)  
doRollSkill @ patching.js:108  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.entityClass.prototype.rollSkill#0 @ libWrapper-wrapper.js:164  
rollSkillMacro @ bundle.min.js:1  
_handleMacros @ bundle.min.js:1  
doHandleActionEvent @ bundle.min.js:1  
handleActionEvent @ bundle.min.js:1  
handleClick @ bundle.min.js:1  
(anonymous) @ bundle.min.js:1  
dispatch @ jquery.min.js:2  
v.handle @ jquery.min.js:2  

14:12:37.026 dae.js:407 Error: Roll.safeEval produced a non-numeric result from expression ""  
    at Function.safeEval (foundry.js:5614)  
    at Roll._evaluateTotal (foundry.js:5540)  
    at Roll._evaluateSync (foundry.js:5527)  
    at Roll.evaluate (foundry.js:5454)  
    at Actor5e.applyDaeEffects (dae.js:403)  
    at Actor5e.applyBaseActiveEffects (dae.js:348)  
    at B.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:551)  
    at Actor5e.🎁CONFIG.Actor.documentClass.prototype.applyActiveEffects#0 (libWrapper-wrapper.js:164)  
    at Actor5e.prepareEmbeddedEntities (foundry.js:12397)  
    at Actor5e.prepareData (foundry.js:9413)  
    at Actor5e.prepareData (entity.js:55)  
    at B.🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:467)  
    at Actor5e.patchedPrepareData (skl-customization.js:16)  
    at B.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:576)  
    at Actor5e.prepareData (dae.js:749)  
    at B.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:576)  
    at Actor5e.🎁CONFIG.Actor.documentClass.prototype.prepareData#0 (libWrapper-wrapper.js:164)  
    at BabeleClientDatabaseBackend._postUpdateDocumentCallbacks (foundry.js:8984)  
    at BabeleClientDatabaseBackend._handleUpdateDocuments (foundry.js:8914)  
    at BabeleClientDatabaseBackend._updateDocuments (foundry.js:8803)  
    at async Function.updateDocuments (document.mjs:366)  
    at async Actor5e.update [as __compendium_tools_original_update] (document.mjs:448)  
    at async ConditionalVisibilityFacade.js:131  
applyDaeEffects @ dae.js:407  
applyBaseActiveEffects @ dae.js:348  
🎁call_wrapper @ libWrapper-wrapper.js:551  
🎁CONFIG.Actor.documentClass.prototype.applyActiveEffects#0 @ libWrapper-wrapper.js:164  
prepareEmbeddedEntities @ foundry.js:12397  
prepareData @ foundry.js:9413  
prepareData @ entity.js:55  
🎁call_wrapped @ libWrapper-wrapper.js:467  
patchedPrepareData @ skl-customization.js:16  
🎁call_wrapper @ libWrapper-wrapper.js:576  
prepareData @ dae.js:749  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.documentClass.prototype.prepareData#0 @ libWrapper-wrapper.js:164  
_postUpdateDocumentCallbacks @ foundry.js:8984  
_handleUpdateDocuments @ foundry.js:8914  
_updateDocuments @ foundry.js:8803  
async function (async)  
_updateDocuments @ foundry.js:8794  
update @ backend.mjs:154  
async function (async)  
update @ backend.mjs:151  
updateDocuments @ document.mjs:366  
update @ document.mjs:448  
callOriginalFunction @ Monkey.js:80  
(anonymous) @ patches.js:100  
setFlag @ document.mjs:596  
(anonymous) @ ConditionalVisibilityFacade.js:131  
hide @ ConditionalVisibilityFacade.js:110  
(anonymous) @ ConditionalVisibilitySystem5e.js:117  
_call @ VM9225:4  
callAll @ foundry.js:153  
callback @ foundry.js:8777  
(anonymous) @ foundry.js:8716  
_handleCreateDocuments @ foundry.js:8716  
_createDocuments @ foundry.js:8612  
async function (async)  
_createDocuments @ foundry.js:8603  
create @ backend.mjs:96  
async function (async)  
create @ backend.mjs:93  
createDocuments @ document.mjs:332  
create @ document.mjs:431  
toMessage @ foundry.js:6128  
toMessage @ d20-roll.js:122  
doRollSkill @ patching.js:114  
async function (async)  
doRollSkill @ patching.js:108  
🎁call_wrapper @ libWrapper-wrapper.js:576  
🎁CONFIG.Actor.entityClass.prototype.rollSkill#0 @ libWrapper-wrapper.js:164  
rollSkillMacro @ bundle.min.js:1  
_handleMacros @ bundle.min.js:1  
doHandleActionEvent @ bundle.min.js:1  
handleActionEvent @ bundle.min.js:1  
handleClick @ bundle.min.js:1  
(anonymous) @ bundle.min.js:1  
dispatch @ jquery.min.js:2  
v.handle @ jquery.min.js:2

ArdonDaros commented:
Thank you for trying to update this mod. I use Pathfinder 1 rules and it appears that none of the functions of the mod work; On latest stable Foundry build.

p4535992 committed:
added package-lock.json

p4535992 committed:
update README.md

p4535992 committed:
abbandoned project

p4535992 committed:
update README.md

p4535992 committed:
update README.md

p4535992 committed:
preparation 0.2.13

p4535992 committed:
version 0.2.13

p4535992 committed:
version 0.3.0

p4535992 committed:
update README.md

p4535992 committed:
update

p4535992 committed:
update README.md

p4535992 committed:
sync version 0.3.1

p4535992 committed:
version 0.3.2

p4535992 committed:
version 0.3.3

p4535992 committed:
changelog update

p4535992 committed:
update README.md

p4535992 committed:
bug fix

p4535992 committed:
version 0.3.4

p4535992 committed:
update README.md

p4535992 committed:
update README.md

p4535992 committed:
version 0.3.5

p4535992 committed:
beta 0.4.15

p4535992 committed:
version 0.4.15 (beta ready)

p4535992 committed:
beta

p4535992 committed:
beta

p4535992 committed:
version 0.4.16

p4535992 committed:
update README.md

p4535992 committed:
a

p4535992 committed:
update hooks

p4535992 committed:
version 0.4.17

p4535992 committed:
version 0.4.18

p4535992 committed:
version 0.4.19

p4535992 committed:
a

p4535992 committed:
version 0.4.20

p4535992 committed:
version 0.4.21

p4535992 committed:
update readme

p4535992 committed:
version 0.4.22

p4535992 committed:
version 0.4.23

p4535992 committed:
version 0.4.24

p4535992 committed:
big update , clean up, rewrite almost everything ?

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
very big clean up for performance and readable of the code

p4535992 committed:
a

p4535992 committed:
prettier

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
version 0.4.25

p4535992 committed:
version 0.4.26

p4535992 committed:
version 0.4.27

p4535992 committed:
many bug fix

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
version 0.4.29

p4535992 committed:
update tutorial

p4535992 committed:
update tutorial.md

p4535992 committed:
update api.md

p4535992 committed:
version 0.4.30

p4535992 committed:
version 0.4.31

p4535992 committed:
update

p4535992 committed:
bug fix

p4535992 committed:
version 0.4.32

p4535992 committed:
version 0.4.33

p4535992 committed:
beta

p4535992 committed:
appleid prettier

p4535992 committed:
version 0.4.34

p4535992 committed:
version 0.4.35

p4535992 committed:
version 0.4.36

p4535992 committed:
update

p4535992 committed:
version 0.4.38

p4535992 committed:
u

p4535992 committed:
update example1.gif

p4535992 committed:
a

p4535992 committed:
some clean up

p4535992 committed:
version 0.5.0

p4535992 committed:
a

p4535992 committed:
b

p4535992 committed:
c

p4535992 committed:
d

p4535992 committed:
version 0.5.1

p4535992 committed:
version 0.5.2

p4535992 committed:
removed forEach on favor of for

p4535992 committed:
set up api

p4535992 committed:
version 0.5.3

p4535992 committed:
a

p4535992 committed:
version 0.5.4

p4535992 committed:
a

p4535992 committed:
a

p4535992 committed:
a

p4535992 committed:
prettier

p4535992 committed:
version 0.5.5

p4535992 committed:
update

p4535992 committed:
a

p4535992 committed:
update README.md

p4535992 committed:
version 0.5.7

p4535992 committed:
version 0.5.8

p4535992 committed:
version 0.5.10

p4535992 committed:
version 0.5.11

p4535992 committed:
update

p4535992 committed:
version 0.5.12

p4535992 committed:
version 0.5.13

p4535992 committed:
version 0.5.14

p4535992 committed:
version 0.5.15

p4535992 committed:
version 0.5.16

p4535992 committed:
version 0.5.17

p4535992 committed:
version 0.5.18

p4535992 committed:
version 0.5.19

p4535992 committed:
add refresh for lighting and sight on perception schedule

p4535992 committed:
bug fix error https://github.com/p4535992/conditional-visibility/issues/23

p4535992 committed:
version 0.5.20

p4535992 committed:
version 0.5.21

p4535992 committed:
update version 0.5.21

p4535992 committed:
rewrite for better performance

p4535992 committed:
big update for performance

p4535992 committed:
u

p4535992 committed:
u

p4535992 committed:
prettier

p4535992 committed:
u

p4535992 committed:
some performance code

p4535992 committed:
preparing version2 of the checker for performance

p4535992 committed:
update checker for better performance

p4535992 committed:
u

p4535992 committed:
prettier

p4535992 committed:
u

p4535992 committed:
u

p4535992 committed:
version 0.5.21 [BREAKING CHANGES]

p4535992 committed:
u

p4535992 committed:
preparation 0.5.22

p4535992 committed:
version 0.5.22

p4535992 committed:
version 0.5.23

p4535992 committed:
version 0.5.24

p4535992 committed:
version 0.5.25

p4535992 committed:
u

p4535992 committed:
version 0.5.26

p4535992 committed:
version 0.5.27 disable hud for loop

p4535992 committed:
version 0.5.28

p4535992 committed:
version 0.5.29

p4535992 committed:
version 0.5.30

p4535992 committed:
version 0.5.31

p4535992 committed:
version 0.5.32

p4535992 committed:
version 0.5.33

p4535992 committed:
u

p4535992 committed:
u

p4535992 committed:
version 0.5.34

p4535992 committed:
version 0.5.35

p4535992 committed:
u

p4535992 committed:
version 0.5.36

p4535992 committed:
u

p4535992 committed:
u

p4535992 committed:
added stealthed

p4535992 committed:
prettier

p4535992 committed:
u

p4535992 committed:
version 0.5.37

p4535992 committed:
version 0.5.38

p4535992 committed:
version 0.5.39

p4535992 committed:
add blinded and blindedOverride conditions

p4535992 committed:
version 0.5.40

p4535992 committed:
update tutorial.md

p4535992 committed:
update effect management

p4535992 committed:
beta

p4535992 committed:
beta

p4535992 committed:
beta

p4535992 committed:
update

p4535992 committed:
beta

p4535992 committed:
update

p4535992 committed:
version 0.5.41

p4535992 committed:
update

p4535992 committed:
version 0.5.42

p4535992 committed:
beta

p4535992 committed:
update package.json

p4535992 committed:
beta

p4535992 committed:
beta

p4535992 committed:
update

p4535992 committed:
removed data flag to hard to manage we read the active effects runtime we lose some performance...

p4535992 committed:
solved issue with updating to 0 the value directly from active effect

p4535992 committed:
solve porlbme with reset ae with value 0

p4535992 committed:
solved issue about the merging of value

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
prettier

p4535992 committed:
solved issue do not delete the active effect when set to 0 the value of active effect change

p4535992 committed:
bug fix

p4535992 committed:
added bug fix

p4535992 committed:
update

p4535992 committed:
we are close to the release

p4535992 committed:
ready for release

p4535992 committed:
version 0.6.0

p4535992 committed:
version 0.6.1

p4535992 committed:
version 0.6.2

p4535992 committed:
version 0.6.3

p4535992 committed:
update

p4535992 committed:
update

p4535992 committed:
version 0.6.4

p4535992 committed:
version 0.6.5

p4535992 committed:
version 0.6.5

p4535992 committed:
Update README.md

p4535992 committed:
update labels

p4535992 committed:
version 0.6.6

p4535992 committed:
version 0.6.6

p4535992 committed:
version 0.6.7

p4535992 committed:
version 0.6.8

p4535992 committed:
version 0.6.9

p4535992 committed:
version 0.6.10

p4535992 committed:
roll eval

p4535992 committed:
version 0.6.11

p4535992 committed:
Update README.md

p4535992 committed:
Update tutorial.md

p4535992 committed:
version 0.6.12

p4535992 committed:
Merge branch 'master' of https://github.com/p4535992/conditional-visibility

p4535992 committed:
pre version 0.6.13

p4535992 committed:
version 0.6.13

p4535992 committed:
u

p4535992 committed:
update

p4535992 committed:
integration pf1

p4535992 committed:
add new module settings for multiselect skills to customize

p4535992 committed:
version 0.6.13

p4535992 committed:
update

p4535992 committed:
version 0.6.15

p4535992 committed:
u

p4535992 committed:
version 0.6.16

p4535992 committed:
u

p4535992 committed:
beta

p4535992 committed:
update .github

Varriount committed:
Update to latest version version of AutoPublish

p4535992 committed:
Merge pull request #54 from Varriount/patch-1

Update to latest version version of AutoPublish

p4535992 committed:
Update README.md
